.github/workflows/storybook.yml

```diff
@@ -0,0 +1,18 @@
+name: Build and Deploy
+on:
+  push:
```

Comment:
where push

Reply:
changed to develop (I'll create this branch later)